README.md

```diff
@@ -1,122 +1,101 @@
+# .NET Interactive Documentation 
```

Comment:
The landing page isn't really documentation. We probably don't need this extra heading.